tests/profiling/collector/test_stack.py

```diff
@@ -246,32 +262,6 @@ def collect(self):
         return []
 
 
-@pytest.mark.skipif(not TESTING_GEVENT, reason="Not testing gevent")
-@pytest.mark.parametrize("ignore", (True, False))
-def test_ignore_profiler_gevent_task(monkeypatch, ignore):
```

Comment:
@P403n1x87 why was this test removed? I'm not sure where in this test gevent actually is introduced so I'm confused about the context of this test...